autobahn_autoreconnect/__init__.py

```diff
@@ -154,7 +154,15 @@ def _create_app_session():
                 return session
 
         self._transport_factory = WampWebSocketClientFactory(_create_app_session, url=self._url, serializers=self._serializers)
-        self._transport_factory.setProtocolOptions(autoPingInterval=5, autoPingTimeout=2)
+
+        autoPingInterval = 10
+        autoPingTimeout = 27
+        if 'autoPingInterval' in self._extra:
```

Comment:
`extra` should be used to forward to the application component. It would make more sense to pass the options directly as named parameters and stored them in `self`.